### ISSUE TITLE
disable toolbar not working

### ISSUE THREAD

**Issue by amit62331** [kolkov/angular-editor#329]
hi @kolkov,
i try to disable toolbar, using enableToolbar: false keyword its not woking.

**Comment by RevathiSekar** [comment]
Code please?

**Comment by amit62331** [comment]
Please find below code and link
https://stackblitz.com/edit/angular-ncwgb2?file=src%2Fapp%2Fapp.component.ts

after "enableToolbar: false", still access the toolbar,  for example "Html code button" it is clickable , so we can copy and paste the html code,
![error](https://user-images.githubusercontent.com/41633112/103151746-e6f6a880-47a6-11eb-9541-c663987561b2.PNG)

**Comment by RevathiSekar** [comment]
You can choose the options in toolbar, but it is not changing any content inside the editor. You can either hide the toolbar using the **showToolbar : true** option or else add the following jquery for restricting and enabling click inside the toolbar

**Comment by prafulhk** [comment]
Yes even i am facing this issue, using enableToolbar: false, stil i can use the toolbar and inside the editor contents are affected based on the selection as well..this option is not working kindly help in this

![image](https://github.com/kolkov/angular-editor/assets/13209620/8b9beb1a-bf86-4a4a-8f95-55591a0a2b35)

![image](https://github.com/kolkov/angular-editor/assets/13209620/c9842fc3-42db-46e4-8439-0c472440544e)
